pydrumscore/export.py

```diff
@@ -528,6 +529,34 @@ def add_note(chord, notedef):
     with open(save_path, "wb") as f:
         f.write(xml_str)
 
+    zip_file= zipfile.ZipFile("pydrumscore/_exported/" + metadata.workTitle + ".mscz", "w")
+    ########################## Export from: export_files
+    # suggestion: Make a little function and make this locally
+    zip_file.write("pydrumscore/_exported/" + filename, filename, compress_type=zipfile.ZIP_DEFLATED)
+    zip_file.write("pydrumscore/export_files/audiosettings.json", "audiosettings.json", compress_type=zipfile.ZIP_DEFLATED)
+    zip_file.write("pydrumscore/export_files/score_style.mss", "score_style.mss", compress_type=zipfile.ZIP_DEFLATED)
+    zip_file.write("pydrumscore/export_files/viewsettings.json", "viewsettings.json", compress_type=zipfile.ZIP_DEFLATED)
+    zip_file.write("pydrumscore/export_files/Thumbnails/thumbnail.png", "Thumbnails/thumbnail.png", compress_type=zipfile.ZIP_DEFLATED)
+
+    ############################ Create container.xml
+    import xml.etree.ElementTree as ET
+    mytree = ET.parse("pydrumscore/export_files/META-INF/container.xml")
+    myroot= mytree.getroot()
+
+    # locate the tag you want to modify
+    for elem in myroot.iter():
+        if elem.tag == "rootfile" and "full-path" in elem.attrib:
```

Comment:
I see in `container.xml` that there are many tags with `rootfile` and `full-path`. So if we rely on an existing tag to change, it will be dependent on the order.

Instead can we just create it on the fly?